Library/Formula/ossp-uuid.rb

```diff
@@ -32,4 +32,20 @@ def install
     system "make"
     system "make", "install"
   end
+
+  patch :DATA
```

Comment:
This line should go above the install block, and the explanatory comments should go above it.